src/apps/api/views/submissions.py

```diff
@@ -212,6 +214,19 @@ def toggle_public(self, request, pk):
         submission.save()
         return Response({})
 
+    @action(detail=True, methods=('POST',))
+    def update_fact_sheet(self, request, pk):
+        request_submission = super().get_object()
+        top_level_submission = request_submission.parent or request_submission
+        # Validate fact_sheet using serializer
+        data = self.get_serializer(top_level_submission).data
+        data['fact_sheet_answers'] = request.data
+        serializer = self.get_serializer(top_level_submission, data=data)
```

Comment:
this seems like some weird indirection, why do you have to do this? I think you can do..

```
        serializer = self.get_serializer(top_level_submission, data=request.data)
```

instead of
```
        data = self.get_serializer(top_level_submission).data
        data['fact_sheet_answers'] = request.data
        serializer = self.get_serializer(top_level_submission, data=data)
```